feat(wbh): add low level doc (#3783)
scaleway/docs-content: managed-services/webhosting/how-to/manage-email-accounts.mdx

  description: Discover how to manage email accounts for Scaleway Web Hosting plans from the console.
  h1: How to manage email accounts
  paragraph: Discover how to manage email accounts for Scaleway Web Hosting plans from the console.
tags: webhosting
dates:
  validation: 2024-10-02
  posted: 2024-10-02
categories:
  - managed-services
---

Send, receive, and store electronic messages through the internet. Email accounts can be accessed via POP3 and IMAP protocols and managed from the Scaleway console.

<Macro id="requirements" />

- A Scaleway account logged into the [console](https://console.scaleway.com)
- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
- [Registered a domain name](/network/domains-and-dns/how-to/register-internal-domain/) at Scaleway or another registrar
- A Web Hosting plan

## How to create an email account

1. Click **Web Hosting** in the **Managed Services** section of the [console](https://console.scaleway.com/) side menu. The **Web Hosting** overview page displays.

> 2. Click <Icon name="seeMore" /> or the domain name of the Web Hosting you want to configure. The **Hosting information** page displays.

Reviewer: ```suggestion
2. Click <Icon name="seeMore" /> or the domain name of the Web Hosting service you want to configure. The **Hosting information** page displays.
```

3. Click the **Emails** tab to display information related to your email accounts.
4. Click **Create email account** to create a new one. A pop-up displays.
5. Select the domain you want to associate the email account with from the drop-down list, then enter a username and password.
    <Message type="tip">
        The username is the part of your email address in front of the @.
    </Message>
6. Click **Create email account** to confirm the action and create the account.

<Message type="tip">
    To access webmail for the email account, click **Access webmail** next to the email account.
</Message>

## How to update the password of an email account

1. Click **Web Hosting** in the **Managed Services** section of the [console](https://console.scaleway.com/) side menu. The **Web Hosting** overview page displays.

> 2. Click <Icon name="seeMore" /> or the domain name of the Web Hosting you want to configure. The **Hosting information** page displays.

Reviewer: ```suggestion
2. Click <Icon name="seeMore" /> or the domain name of the Web Hosting service you want to configure. The **Hosting information** page displays.
```

3. Click the **Emails** tab to display information related to your email accounts.
4. Click <Icon name="more" /> > **Change password** next to the email account whose password you want to change. A pop-up displays.
5. Enter the new password and click **Change password** to submit the form and update the password.

## How to delete an email account

1. Click **Web Hosting** in the **Managed Services** section of the [console](https://console.scaleway.com/) side menu. The **Web Hosting** overview page displays.

> 2. Click <Icon name="seeMore" /> or the domain name of the Web Hosting you want to configure. The **Hosting information** page displays.

Reviewer: ```suggestion
2. Click <Icon name="seeMore" /> or the domain name of the Web Hosting service you want to configure. The **Hosting information** page displays.
```

3. Click the **Emails** tab to display information related to your email accounts.
4. Click <Icon name="more" /> > **Delete** next to the email account you want to delete. A pop-up displays.

> 5. Type **DELETE** in the popup and click **Delete email account** to confirm the action and delete the account.

Reviewer: ```suggestion
5. Type **DELETE** in the pop-up and click **Delete email account** to confirm the action and delete the account.
```
